sym/484a75a6-6a27-4499-bfb3-77ea90f6b189/symbol.lp

```diff
@@ -0,0 +1,62 @@
+(librepcb_symbol 484a75a6-6a27-4499-bfb3-77ea90f6b189
+ (name "Thermistor")
+ (description "")
+ (keywords "")
+ (author "J. R. Schmid")
+ (version "0.1")
+ (created 2020-05-10T13:17:25Z)
+ (deprecated false)
+ (category 1039f038-20a6-4bfe-89c1-99f34fbb45bd)
+ (pin 5d2e8d52-8e00-4cc2-b6ca-bcd5ca811b83 (name "1")
+  (position -5.08 0.0) (rotation 0.0) (length 2.0)
+ )
+ (pin 7248ab54-d225-4fcd-8f3a-f270efa20a82 (name "2")
+  (position 5.08 0.0) (rotation 180.0) (length 2.0)
+ )
+ (polygon 5eef6e59-207d-452d-8c96-40d36bce97b6 (layer sym_outlines)
+  (width 0.254) (fill false) (grab_area true)
+  (vertex (position -3.08 -1.016) (angle 0.0))
+  (vertex (position -3.08 1.016) (angle 0.0))
+  (vertex (position 3.08 1.016) (angle 0.0))
+  (vertex (position 3.08 -1.016) (angle 0.0))
+  (vertex (position -3.08 -1.016) (angle 0.0))
+ )
+ (polygon c375f034-014a-4e5e-b3fa-7d5375b29190 (layer sym_outlines)
+  (width 0.1) (fill false) (grab_area false)
+  (vertex (position 3.15 2.0) (angle 0.0))
+  (vertex (position 1.8 2.0) (angle 0.0))
+  (vertex (position -1.8 -2.0) (angle 0.0))
+  (vertex (position -3.15 -2.0) (angle 0.0))
+ )
+ (polygon 3aad57d2-ce6a-4a4e-b669-7430ccecd2e4 (layer sym_outlines)
+  (width 0.1) (fill false) (grab_area false)
+  (vertex (position -4.4 2.54) (angle 0.0))
+  (vertex (position -1.4 2.54) (angle 0.0))
+ )
+ (polygon cee832ee-5c28-4ab5-8de8-3f0b0975676d (layer sym_outlines)
+  (width 0.1) (fill false) (grab_area false)
+  (vertex (position -2.2 3.0) (angle 0.0))
+  (vertex (position -1.4 2.54) (angle 0.0))
+  (vertex (position -2.2 2.08) (angle 0.0))
+ )
+ (polygon 8e6366f9-6160-4bc0-9efb-1a8cb1932549 (layer sym_outlines)
+  (width 0.1) (fill false) (grab_area false)
+  (vertex (position -1.4 1.8) (angle 0.0))
+  (vertex (position -4.4 1.8) (angle 0.0))
+ )
+ (polygon 14836244-597f-4bd4-b796-932fcf52e3f6 (layer sym_outlines)
+  (width 0.1) (fill false) (grab_area false)
+  (vertex (position -3.6 2.26) (angle 0.0))
+  (vertex (position -4.4 1.8) (angle 0.0))
+  (vertex (position -3.6 1.34) (angle 0.0))
+ )
+ (text 58bec4c1-dc43-4d52-8987-f717d925f19c (layer sym_names) (value "{{NAME}}")
+  (align left bottom) (height 2.54) (position -3.08 3.0) (rotation 0.0)
+ )
+ (text acf4f029-f5e3-468b-ae38-95a59b027579 (layer sym_values) (value "{{RESISTANCE}}")
```

Comment:
The value should not be hardcoded. Use `{{VALUE}}` in the symbol, and `{{RESISTANCE}}` as the default value in the component.

```suggestion
 (text acf4f029-f5e3-468b-ae38-95a59b027579 (layer sym_values) (value "{{VALUE}}")
```

Reply:
> Use {{VALUE}} in the symbol

Done.

> and {{RESISTANCE}} as the default value in the component

Not sure if I understand your meaning. So, in the component, table "attributes", the _Key_ should be "VALUE", the _Type_ should be "Resistance", and the _Value_ should be "RESISTANCE"?

Reply:
Changed, I hope.

Reply:
You can use the resistor as reference:

![2020-06-15-140732_1331x373_scrot](https://user-images.githubusercontent.com/105168/84655633-95c14e80-af11-11ea-8c0d-778d0831f546.png)

It has a RESISTANCE attribute (on the left), and uses that attribute as default value (on the right).